packages/ckeditor5-engine/tests/view/domconverter/dom-to-view.js

```diff
@@ -158,6 +158,42 @@ describe( 'DomConverter', () => {
 			expect( viewFragment2 ).to.equal( viewFragment );
 		} );
 
+		it( 'should assign $rawContent custom property for view elements registered as raw content elements', () => {
```

Comment:
Too few tests IMO. Right now it tests only one pattern. I think that we could check a bit more:

*   multiple patterns registered
*   what if `div.raw-content-container` has other attributes, classed or styles, ie `div[ data-foo="bar"].raw-content-container`